assesspy/tests/conftest.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+import pandas as pd
+import pytest as pt
+
+import assesspy as ap
+
+
+@pt.fixture(autouse=True, scope="class")
```

Comment:
**[Nitpick, non-blocking]** It doesn't really matter much, but any reason not to set the seed at the `session` level? Are there some classes or modules that might want to use a different seed?

```suggestion
@pt.fixture(autouse=True, scope="session")
```

Reply:
Good thinking!